src/shared/meshes/cell_linked_list.hpp

```diff
@@ -44,4 +44,20 @@ void CellLinkedList::searchNeighborsByParticles(
                  });
 }
 //=================================================================================================//
+template <class LocalDynamicsFunction>
```

Comment:
I set this function as a member function for now, but I can also move this part to `InteractionSplit` if you think that is better

Reply:
Keep as a member function of `CellLinkedList` for now